shell/platform/android/io/flutter/app/FlutterActivityDelegate.java

```diff
@@ -141,7 +141,7 @@ public void onCreate(Bundle savedInstanceState) {
       Window window = activity.getWindow();
       window.addFlags(LayoutParams.FLAG_DRAWS_SYSTEM_BAR_BACKGROUNDS);
       window.setStatusBarColor(0x40000000);
-      window.getDecorView().setSystemUiVisibility(PlatformPlugin.DEFAULT_SYSTEM_UI);
+      WindowCompat.setDecorFitsSystemWindows(window, false);
```

Comment:
This line means edge-to-edge mode will become the default mode (related to flutter/flutter#86248 ?). I would be glad if we can land this, but do note that this will be a breaking change for apps that don't handle paddings/safe area properly. My team had to manually apply non-trivial fixes (e.g. cl/397106186 and cl/397283284) before enabling this.